### PR TITLE
Accept relaxed variations when expanding short props

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes spring-cloud/spring-cloud-dataflow#779

### CODE REVIEW

File: spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java [hunk 1]
```diff
@@ -121,8 +122,8 @@
 	 * @param commonProperties         common set of application properties
 	 */
 	public StreamDeploymentController(StreamDefinitionRepository repository,
-			DeploymentIdRepository deploymentIdRepository, AppRegistry registry, AppDeployer deployer,
-			ApplicationConfigurationMetadataResolver metadataResolver, CommonApplicationProperties commonProperties) {
+	                                  DeploymentIdRepository deploymentIdRepository, AppRegistry registry, AppDeployer deployer,
+	                                  ApplicationConfigurationMetadataResolver metadataResolver, CommonApplicationProperties commonProperties) {
```
Comment (jvalkeal) [inline spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java]: Spaces to tabs

File: spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java [hunk 2]
```diff
@@ -324,8 +332,8 @@ private void assertNoAmbiguity(List<ConfigurationMetadataProperty> longForms) {
 	}
 
 	private void parseAndPopulateProperties(Map<String, String> streamDeploymentProperties,
-			Map<String, String> appDeploymentProperties, String producerPropertyPrefix, String consumerPropertyPrefix,
-			String appPrefix) {
+	                                        Map<String, String> appDeploymentProperties, String producerPropertyPrefix, String consumerPropertyPrefix,
+	                                        String appPrefix) {
```
Comment (jvalkeal) [inline spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java]: Spaces to tabs

**Comment by jvalkeal** [comment]
Other than that looks good as all combinations worked.

File: spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java (hunk 1 above)
Comment (ericbottard) [inline spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java]: Don't tell me IDEA lost my formatting prefs on upgrade...

**Comment by ericbottard** [comment]
Force pushed formatting changes

**Pull request merged**